Bottom: 7ad3bfc91ec638dca0d9ba315b8c2ae92a6d5e89
Top:    712f17084e35117b8dda0a7a32b3b7df0eecfd54
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:59 -0500

Create certificate directories during image build


---
diff --git a/docker/docker-svn/Dockerfile b/docker/docker-svn/Dockerfile
index 967bca5..7397d68 100644
--- a/docker/docker-svn/Dockerfile
+++ b/docker/docker-svn/Dockerfile
@@ -10,6 +10,7 @@ MAINTAINER Joseph Lutz <Joseph.Lutz@novatechweb.com>
 RUN apt-get update && \
   DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
     apache2 \
+    ca-certificates \
     enscript \
     libapache2-svn \
     php5-common \
diff --git a/docker/docker-svn/configure.sh b/docker/docker-svn/configure.sh
index fab90ad..bff6502 100755
--- a/docker/docker-svn/configure.sh
+++ b/docker/docker-svn/configure.sh
@@ -6,6 +6,10 @@ SVN_BASE_DIR=/var/lib/svn
 # Create a directory for a dead symlink
 mkdir /var/cache/websvn/tmp
 
+# Create certificate directories
+mkdir -p /etc/grid-security/certificates
+mkdir -p /etc/ssl/private
+
 # comment out apache2 config file lines that reference the environment variables
 sed -i 's|^Mutex file|#Mutex file|' /etc/apache2/apache2.conf
 sed -i 's|^PidFile |#PidFile |' /etc/apache2/apache2.conf
